src/App.js

```diff
@@ -1,14 +1,20 @@
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
 import {Navbar,Footer} from "./components/components";
-// import Footer from "./components/Footer";
```

Comment:
i will suggest you to make seperate folder for layout component like navbar footer so files in one folder will reduce

Reply:
Yup will do that.